src/common.ts

```diff
@@ -0,0 +1,33 @@
+import {TabsController} from './runtime/TabsController';
+
+export const TABS_CLASSNAME = 'diplodoc-tabs';
+export const TABS_LIST_CLASSNAME = 'diplodoc-tab-list';
+export const TAB_CLASSNAME = 'diplodoc-tab';
+export const TAB_PANEL_CLASSNAME = 'diplodoc-tab-panel';
```

Comment:
I decided to rename class names to avoid effect from old controller. If you use any client js from `ym-transform` you will always embed all runtime code for all plugins. 
It is not a breaking changes because this modules was not published yet.